features/cli/security_status.feature

```diff
@@ -964,6 +964,6 @@ Feature: CLI security-status command
       """
 
     Examples: ubuntu release
-      | release | machine_type  | pkg_in_updates          | pkg_in_security      |
-      | xenial  | lxd-container | base-files=9.4ubuntu4   | wget=1.17.1-1ubuntu1 |
-      | noble   | lxd-container | xxd=2:9.1.0016-1ubuntu7 | less=590-2ubuntu2    |
+      | release | machine_type  | pkg_in_updates               | pkg_in_security      |
+      | xenial  | lxd-container | base-files=9.4ubuntu4        | wget=1.17.1-1ubuntu1 |
+      | noble   | lxd-container | xz-utils=5.6.1+really5.4.5-1 | less=590-2ubuntu2    |
```

Comment:
I wonder if we can pick a package which has less chance of getting security updates at least soon, and `xz` scares me a bit.
However, I picked `xxd` and now here we are.
I think this is good and we fix the test again when needed.